Source/WebCore/css/FontFaceSource.idl

```diff
@@ -27,3 +27,7 @@
 interface mixin FontFaceSource {
     readonly attribute FontFaceSet fonts;
 };
+
+// FIXME: Missing following from standard.
+// Document includes FontFaceSource;
+// WorkerGlobalScope includes FontFaceSource;
```

Comment:
These are not missing, they're in Document.idl / WorkerGlobalScope.idl. I would remove this comment.

Reply:
Fixed in 'd99f558' commit.